yt/geometry/selection_routines.pyx

```diff
@@ -703,9 +703,40 @@ cdef class SelectorObject:
                 ("domain_width[1]", self.domain_width[1]),
                 ("domain_width[2]", self.domain_width[2]))
 
+    def _get_state_list(self):
+        # return a list of attrs needed for __setstate__: implement for each subclass
+        if isinstance(self,SelectorObject):
```

Comment:
I am not sure I understand this.  In what instances will this not return True?  Subclasses should return True here.

```
In [1]: class A: pass

In [2]: class B(A): pass

In [3]: c = B()

In [4]: isinstance(c, A)
Out[4]: True
```

Reply:
oh right, ya, that's wrong.